src/dotnet/commands/dotnet-msbuild/Program.cs

```diff
@@ -1,12 +1,16 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
 
 namespace Microsoft.DotNet.Tools.MSBuild
 {
     public class MSBuildCommand
     {
         public static int Run(string[] args)
         {
+            // Ignore Ctrl-C for the remainder of the command's execution
```

Comment:
Whoops, this one can be removed.  It's already covered by the one in `MSBuildForwardingApp`.